medusa/notifiers/pushover.py

```diff
@@ -65,31 +65,21 @@ def _sendPushover(self, msg, title, sound=None, userKey=None, apiKey=None, prior
         # build up the URL and parameters
         msg = msg.strip()
 
-        # send the request to pushover
+        # default args
```

Comment:
I suggest also updating `setup.cfg` and removing linting rules ignores for this file if there are any